catalogue/webapp/components/IIIFViewer/MultipleManifestList.tsx

```diff
@@ -20,12 +20,15 @@ const MultipleManifestListPrototype: FunctionComponent = () => {
         {parentManifest?.items.map((manifest, i) => (
           <li key={manifest.id}>
             <NextLink
-              {...itemLink({
-                workId: work.id,
-                langCode: lang,
-                manifest: i + 1,
-                canvas: 1,
-              })}
+              {...itemLink(
+                {
+                  workId: work.id,
+                  langCode: lang,
+                  manifest: i + 1,
+                  canvas: 1,
+                },
+                'manifests_navigation'
+              )}
```

Comment:
One of the benefits of the newer `itemLink` helpers are that they enforce tracking in the type system – we have to provide a source for the links.